### PR TITLE
Remove allocated family fee

### PR DESCRIPTION
**Pull request by twinsfan421**
## 💅🏼 What issue does this fix?
#2550 This will remove the column allocated_family_fee from the nebraska_approval_amounts table. 

## 🍂 Before You Submit

* [x] Did you write & run tests and linters?
* [ ] Did you run Google Lighthouse and/or WebAIM (Wave) on UI components in your PR?
* [ ] Does your PR contain any required translations?
* [ ] Are your primary keys UUIDs on any new tables?

## 🛷 Deployment

* [x] Migrations
* [ ] Dependencies

## 🧵 Steps to set up locally

Run bundle exec rails db:migrate

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@
     t.index ["expires_on"], name: "index_approvals_on_expires_on"
   end
 
-  create_table "attendances", id: :uuid, default: -> { "public.gen_random_uuid()" }, force: :cascade do |t|
```
Comment (katelovescode) [inline db/schema.rb:36]: @twinsfan421 can you double check your postgres version and try and run `bundle exec rails db:migrate` again?  This schema prefix removal might be a side effect of a different postgres version.  I'm running Postgres 13 using Postgres.app on my MacBook Pro.  I did about 5 mins of research and couldn't find a quick answer to which version or configuration retains the prefixes. If you can't find a way to prevent it, could you remove these changes from your commit?

File: db/migrate/20220608035549_remove_allocated_family_fee_from_nebraska_approval_amounts.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class RemoveAllocatedFamilyFeeFromNebraskaApprovalAmounts < ActiveRecord::Migration[6.1]
+  def up
+    remove_column :nebraska_approval_amounts, :allocated_family_fee
+  end
+
+  def down
+    add_column :nebraska_approval_amounts, :allocated_family_fee, :decimal
+  end
```
Comment (katelovescode) [inline db/migrate/20220608035549_remove_allocated_family_fee_from_nebraska_approval_amounts.rb:8]:
```suggestion
  def change
    remove_column :nebraska_approval_amounts, :allocated_family_fee, :decimal
  end
```
if you add the column type, this should be reversible

File: spec/fixtures/files/wonderschool_necc_onboarding_data.csv [hunk 3]
```diff
@@ -1,6 +1,6 @@
...
-Jacob Ford,Jacob,Ford,31610139,Rebecca's Childcare,2017-09-01,No,14635435,,276,1656,25,2020-09-01,2021-08-31,No,,,rebecca@rebecca.com,68845,Corke,Family Child Care Home II,Step 4,Yes,0.00,2020-09-01,2021-08-31,0.00,,,,,,,,,,,,,,,,
-Linda Rogers,Linda,Rogers,15228275,Rebecca's Childcare,2019-02-21,No,14635435,,276,1656,22,2020-09-01,2021-08-31,No,,,rebecca@rebecca.com,68845,Corke,Family Child Care Home II,Step 4,Yes,0.00,2020-09-01,2021-08-31,0.00,,,,,,,,,,,,,,,,
-Paz Korman,Paz,Korman,81185504,Rebecca's Childcare,2017-05-17,No,12312445,,138,828,33,2021-01-29,2021-07-31,No,,,rebecca@rebecca.com,68845,Corke,Family Child Care Home II,Step 4,Yes,108.00,2021-01-29,2021-07-31,108.00,,,,,,,,,,,,,,,,
-Becky Falzone,Becky,Falzone,69370816,Kate's Kids,2013-12-26,No,56582912,37827,330,1760,45,2020-11-24,2021-11-23,Yes,90.77,9.43,kate@kate.com,68845,Corke,Family Child Care Home I,Step 5,No,60.00,2020-11-24,2021-05-23,60.00,85.00,2021-05-24,2021-11-23,85.00,,,,,,,,,,,,
```
Comment (katelovescode) [inline spec/fixtures/files/wonderschool_necc_onboarding_data.csv:6]: Can you revert this change?  Our spreadsheet still has this column in it, and we want to make sure that retaining the column doesn't cause any of our behavior to break.

File: spec/fixtures/files/wonderschool_necc_onboarding_renewal_data.csv [hunk 4]
```diff
@@ -1,3 +1,3 @@
...
-Thomas Eddleman,Thomas,Eddleman,14047907,Rebecca's Childcare,2010-09-01,No,14635435,37821,276,1656,30,2021-09-01,2022-08-31,No,,,rebecca@rebecca.com,68845,Corke,Family Child Care Home II,Step 4,Yes,0.00,2020-09-01,2021-08-31,0.00,120.00,2021-09-01,2022-08-31,120.00,,,,,,,,,,,,
\ No newline at end of file
+Full Name,First Name,Last Name,Client ID,Provider Name,Date of birth (required),Enrolled in School (Kindergarten or later),Case number,Wonderschool ID,Authorized full day units,Authorized hourly units,Authorized weekly hours,Effective on,Expires on,Special Needs Rate?,Special Needs Daily Rate,Special Needs Hourly Rate,Provider Email,Business Zip Code,Business County,Business License,Business QRIS rating,Accredited,Approval #1 - Family Fee,Approval #1 - Begin Date,Approval #1 - End Date,Approval #2 - Family Fee,Approval #2 - Begin Date,Approval #2 - End Date,Approval #3 - Family Fee,Approval #3 - Begin Date,Approval #3 - End Date,Approval #4 - Family Fee,Approval #4 - Begin Date,Approval #4 - End Date,Approval #5 - Family Fee,Approval #5 - Begin Date,Approval #5 - End Date
```
Comment (katelovescode) [inline spec/fixtures/files/wonderschool_necc_onboarding_renewal_data.csv:1]: Ditto here - let's leave this file as is.

**Review by katelovescode — CHANGES_REQUESTED** [review]
Couple tweaks requested!

File: db/schema.rb (hunk 1 above)
Comment (twinsfan421) [reply db/schema.rb]: @katelovescode  I just pushed up the fixes.  Thanks

**Review by katelovescode — APPROVED** [review]
